### ISSUE TITLE
Check that the URL used to call MLS for the search region is correct

### ISSUE THREAD

**Issue by jwhitlock** [mozilla-mobile/android-components#8412]
As companion work for issue #8411, I want to confirm that the Mozilla Location Service (MLS) is being called correctly by the released Fenix client.

The MLS URL is constructed in two parts. The first is a constant:

https://github.com/mozilla-mobile/android-components/blob/e2c0fd0376e71915c0669875d9872a3234a068bb/components/service/location/src/main/java/mozilla/components/service/location/MozillaLocationService.kt#L27

The rest of the URL is combined with an API key:

https://github.com/mozilla-mobile/android-components/blob/e2c0fd0376e71915c0669875d9872a3234a068bb/components/service/location/src/main/java/mozilla/components/service/location/MozillaLocationService.kt#L56

This creates a URL like https://location.services.mozilla.com/v1/country?key=APIKEY , which can be used to determine the client's country / region based on the IP address.

In my notes, I have a key for Fenix (starts with `8a4c`). Its use peaked around July 1, and dropped to a low level around July 20. I would expect growth around August 27, but it isn't there.

There are a wide variety of user agents associated with that API key. For September 15, the most common one was `MozAC/49.0.20200702190156`. Another that stands out is `MozAC/42.0.1`, because it calls the path `/data:text/plain,/country?key=8a4c...`, which is wrong in interesting ways.

@st3fan, can you determine the API key used in release, and either post the prefix here or send to me via another channel? With the correct key, I can check from the MLS side if requests are completing, how long they are taking, etc.

I was previously concerned about API-based rate limiting, but we do not rate limit the `/v1/country` API.

**Comment by jwhitlock** [comment]
Thanks, I have confirmation from the team that ``8a4c`` is the correct key.

A top requester for MLS on September 15, about 500k requests, has user agent "MozAC/46.0.0" (AC = Android Components?), that ``POST``ing to  https://location.services.mozilla.com/v1/country?key= (not a typo, there is no API key in the request). Is there a chance this is the released version, and it has a blank MLS API key?

I do see a smaller volume of requests, below 400 each, from later versions, like `MozAC/51.0.0`.

**Comment by st3fan** [comment]
For completeness, we verified the following APKs:

```
firefox_v80.1.2_arm.apk
firefox_v80.1.2_arm64.apk
firefox_v80.1.2_x86.apk
firefox_v80.1.2_x86_64.apk
```

**Comment by jwhitlock** [comment]
We've successfully confirmed the MLS key for Fenix, I support closing this issue as fixed.

Currently, Fenix doesn't call MLS at all (https://github.com/mozilla-mobile/fenix/pull/15161), which will hopefully change in a future release.

The `MozAC/46.0.0` calls are from a different, unknown product, which may not be a Mozilla product. There is not enough data on the MLS side to tell for sure. A fix for #8421 could help determine the calling product, once everyone updates.
